website/src/pages/index.md

```diff
@@ -1,11 +1,6 @@
-# Component Library
+# Europa Component Library
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut convallis
-metus non lectus dignissim, ac viverra elit convallis. Pellentesque
-tempus purus id facilisis lacinia.
+The Europa Component Library (ECL) is a comprehensive style guide containing all the design elements and visual standards that make up the European Commission websites hosted under the ec.europa.eu domain and the European Union website hosted under the europa.eu domain.
 
-Nulla consequat suscipit commodo. Fusce sollicitudin orci id volutpat
-tempor. Duis vestibulum, diam vel gravida dapibus, odio velit eleifend
-felis, id elementum dolor nisi id arcu.
-
-Select between the available libraries:
+The European Commission’s Digital transformation team (DTT) is the owner of this library. Please contact COMM Europa Management (CEM) for support on using this resource for a European Commission website.
+To start building webpage using the ECL components choose one the component libraries below.
```

Comment:
Just to make sure: if you want `To start building...` to be in a new paragraph, you need to add an extra blank line between this paragraph and the previous one. Otherwise, please ignore this comment.